src/components/Header/components/Logo/Logo.tsx

```diff
@@ -0,0 +1,11 @@
+import logo1 from 'images/logo1.png';
+
+export const Logo = () => {
+  return (
+    <img
+      style={{ width: '111px', height: '48px', top: '15px', left: '1270px' }}
```

Comment:
Inline styles are not recommended. Please use a class name instead.

Reply:
Fixed